www/modules/custom/borg_cleanup/borg_cleanup.module

```diff
@@ -58,4 +58,35 @@ function borg_cleanup_form_user_profile_form_alter(&$form, &$form_state) {
   unset($form['account']['pass']['#description']);
   $form['account']['mail']['#description'] = str_replace('A valid e-mail address. All e-mails from the system will be sent to this address. ', '', $form['account']['mail']['#description']);
   $form['account']['mail']['#description'] = str_replace(' or wish to receive certain news or notifications by e-mail', '', $form['account']['mail']['#description']);
+
+  $form['email-subscriptions'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Email Subscriptions'),
+    '#weight' => 22,
+    '#collapsible' => TRUE,
+  );
+
+  $account = $form['#user'];
+  $borg_security_list = $form['field_security_mailing_list'];
+  unset($form['field_security_mailing_list']);
+
+  $contact_field = $form['contact']['contact'];
+  unset($form['contact']);
+
+  $form['email-subscriptions']['contact'] = $contact_field;
+  $form['email-subscriptions']['borg-mailers-security'] = $borg_security_list;
+}
```

Comment:
I think the additional variables `$borg_security_list` and `$contact_field` might be obscuring what's going on here instead of clarifying. Can we manipulate the form directly, instead?

```php
$form['email-subscriptions']['contact'] = $form['contact']['contact'];
unset($form['contact']);
$form['email-subscriptions']['borg-mailers-security'] = $form['field_security_mailing_list'];
unset($form['field_security_mailing_list']);
```